app/javascript/lib/scp-api.js

```diff
@@ -61,6 +61,19 @@ function defaultPostInit(mock=false) {
   return init
 }
 
+/**
+ * Retrieve all study files for a given study
+ * Docs: https://singlecell.broadinstitute.org/single_cell/api/v1#/StudyFiles/study_study_files_path
+ *
+ * @param {String} studyId: BSON ID of study
+ * @param {Boolean} mock If using mock data.  Helps development, tests.
+ */
+export async function fetchStudyFiles(studyId, mock=false) {
+  const path = `/studies/${studyId}/study_files`
```

Comment:
This endpoint actually isn't used anywhere... I had almost forgotten that I put it in.  The original implementation of this work relied on making an API call to get the list of current study files, but that proved to not be very performant when rendering the select input for raw counts associations (largely because React renders everything twice when calling `useState`).  I pivoted away from this implementation in favor of the `window.SCP.currentStudyFiles` approach, but didn't go back to remove this method.

All that being said, I agree with the comments, however I'm going to defer that work in favor of removing this unused method.  If and when we come back to adding more methods to `scp-api.js`, then I would propose we do that refactoring then.

Reply:
Sounds good!  Removing this for now would address my concern.

Reply:
a23626f